test/e2e/storage/csi_objects.go

```diff
@@ -152,21 +211,20 @@ func csiClusterRoleBindings(
 		bindingString = "Unbinding"
 	}
 	By(fmt.Sprintf("%v cluster roles %v to the CSI service account %v", bindingString, clusterRolesNames, sa.GetName()))
-	clusterRoleBindingClient := client.RbacV1().ClusterRoleBindings()
+	clusterRoleBindingClient := client.RbacV1beta1().ClusterRoleBindings()
```

Comment:
why beta?

Reply:
I can change this back to v1. There was a bug at some point I thought was due to podsecuritypolicy being only supported in beta